specs/git/update_git_index_after_ignoring_files.yaml

```diff
@@ -0,0 +1,23 @@
+---
+# The name of the workflow.
+name: Delete newly git-ignored files from your repository
+# The corresponding command for the workflow. Any arguments should be surrounded with two curly braces. E.g `command {{arg}}`.
+command: |-
+  git rm -r --cached .
+  git add .
+# Any tags that the workflow should be categorized with.
+tags:
+  - git
+  - gitignore
+  - version control
```

Comment:
in our experience, the best workflows have fewer tags (see https://github.com/warpdotdev/workflows#what-makes-a-useful-workflow). Given that--could we remove the `gitignore` and `version control` flags?

Reply:
Done!